actions/hv.patch.reboot.yaml

```diff
@@ -0,0 +1,28 @@
+---
+description: Patch and Reboot a hypervisor
+enabled: true
+entry_point: src/openstack_actions.py
+name: hv.patch.reboot
+parameters:
+  name:
+    type: string
+    description: Name of host in Icinga
+    required: true
```

Comment:
We could remove this parameter as the hypervisor hostname is used as the name in icinga